internal/generator/helpers/optional_pair.go

```diff
@@ -0,0 +1,29 @@
+package helpers
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type OptionalPair struct {
+	key   string
+	Value interface{}
```

Comment:
lowercase v?

Reply:
I remembered it is public because it is being set in various places after being "new'd".  We do not have need to expose the key but maybe the semantics should be to create SetValue but that seems unnecessary